### DISCUSSION TITLE
Sample log file for net_connection_lnx_back_connect_shell_dev.yml

### DISCUSSION THREAD

**Discussion by arorabharat144** [SigmaHQ/sigma#5019] [Q&A] [answered]
Can anyone provide a sample log file that would match on net_connection_lnx_back_connect_shell_dev.yml sigma rule? Thanks in advance!

**Comment by nasbench** [comment, marked as answer]
https://github.com/SigmaHQ/sigma/commit/ae22f2ae7eaaaa3d0a4b8b5fdc138a8bc97e5fab#commitcomment-147014979
